mne/preprocessing/maxwell.py

```diff
@@ -84,12 +87,12 @@ def maxwell_filter(raw, origin=(0, 0, 40), int_order=8, ext_order=3,
 
     if raw.proj:
         raise RuntimeError('Projectors cannot be applied to raw data.')
-    if 'dev_head_t' not in raw.info.keys():
-        raise RuntimeError("Raw.info must contain 'dev_head_t' to transform "
-                           "device to head coords")
```

Comment:
This should never happen, if it ever did it was a failure of raw data reading on our part. I think at one point it could have happened, but not anymore.